tools/tick-cluster.js

```diff
@@ -517,6 +536,7 @@ function main() {
 function displayMenu(logFn) {
     logFn('\td <flag>\tSet debug flag');
     logFn('\tD\t\tClear debug flags');
+    logFn('\tG\t\tStop gossip');
```

Comment:
Yes, maybe we should change the casing. Because normally the action is initiated with the lower case and after that reverted with the upper case. What do you think about following that paradigm as well?